_data/tool_and_resource_list.yml

```diff
@@ -2660,3 +2660,48 @@
   id: better-bibtex
   name: Better BibTeX (BBT)
   url: https://retorque.re/zotero-better-bibtex/
+- description: GA4GH frames policy and builds standards to meet the real-world needs of the global genomics and health community.
+  id: ga4gh
+  name: Global Alliance for Genomics and Health (GA4GH)
```

Comment:
See also my point about projects below. I would say that organisations such as GA4GH do not fit the table of resources. The table should contain relevant outputs of such organisations instead. Let us discuss @bedroesb